lib/rust/flexer/src/automata/symbol.rs

```diff
@@ -31,19 +31,20 @@ impl Default for Symbol {
 }
 
 impl From<u32> for Symbol {
-    fn from(val:u32) -> Symbol {
-        Symbol{val}
+    fn from(value:u32) -> Symbol {
+        Symbol { value }
```

Comment:
You are using 3 styles in your code so far:
1. `Symbol{value}`
2. `Symbol {value}`
3. `Symbol { value }`

Accoridng to our style guide,e the first one is correct, as we treat `{}` just like `()` (fn call)